pkg/runtime/wfengine/workflow.go

```diff
@@ -528,8 +546,13 @@ func (wf *workflowActor) saveInternalState(ctx context.Context, actorID string,
 
 func (wf *workflowActor) createReliableReminder(ctx context.Context, actorID string, namePrefix string, data any, delay time.Duration) (string, error) {
 	// Reminders need to have unique names or else they may not fire in certain race conditions.
-	reminderName := fmt.Sprintf("%s-%s", namePrefix, uuid.NewString()[:8])
-	wfLogger.Debugf("%s: creating '%s' reminder with DueTime = %s", actorID, reminderName, delay)
+	var reminderName string
+	if strings.Contains(namePrefix, "timer") {
+		reminderName = fmt.Sprintf("timer-%s", uuid.NewString()[:8])
```

Comment:
Please don't use uuid.NewString() which can panic. Instead, use uuid.NewRandom() (and handle errors)

Reply:
Please make magic number 8 a constant.

Reply:
Why is it that this namePrefix is a special case which requires the reminder name to just timer instead of the incoming name prefix?